site/_posts/2018-04-04-quandries.md

```diff
@@ -0,0 +1,33 @@
+---
+title: quandries
```

Comment:
Is the misspelling intentional? I never know with you @d-m-u ;)

Also, I'd still prefer if the blog post title is prefaced with "Last Month in ManageIQ". Thanks!